pom.xml

```diff
@@ -2,12 +2,12 @@
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>com.avides.springboot.testcontainer</groupId>
```

Comment:
springtainer

Reply:
Changed group id to `com.avides.springboot.springtainer`.